README.md

```diff
@@ -1,2 +1,25 @@
-# community
-Repositorio con temas centrales respecto a la comunidad
+# DEVZ Community
+###### Repositorio con temas centrales respecto a la comunidad
+
+<div align="center" >
+  <img src="https://devz.mx/content/images/2018/09/DEVZ_Logo-05.png" alt="devz_logo"  width="150" margin="">
+</div>
+
+#### 👋 Hola y bienvenido a la comunidad de desarrolladores que harán que crezcas exponencialmente en tu carrera como dev.
+
+Devz es una comunidad donde desarrolladores de distintos niveles se ayudan entre si para progresar en su vida profesional.
+
+Encuéntranos en:
+
+- [Facebook](https://www.facebook.com/DevzCommunity)
+- [Blog](https://devz.mx/)
+- [Twitter](https://twitter.com/DevzCommunity)
+
+Unete a la conversación en el [Slack de Devz Community](https://slack.devz.mx) y revisa tus temas preferidos en nuestros [canales](https://github.com/devzcommunity/community/blob/master/SLACK_INDEX.md).
+
+Ademas:
+
+- 🎙 Escucha a las voces de los DEVZ en los [podcast](https://github.com/devzcommunity/community/blob/master/PODCASTS.md) de la comunidad.
```

Comment:
nit: `en los podcasts de la comunidad`, podcasts in plural

Reply:
Incluyendo la sugerencia de @tonymtz y siguiendo con mis comentarios sobre lenguaje inclusivo, que les parece este cambio a la frase:

```suggestion
- 🎙 Escucha algunas de nuestras voces en los [podcasts](https://github.com/devzcommunity/community/blob/master/PODCASTS.md) de la comunidad.
```